app/views/spotlight/search_configurations/_sort.html.erb

```diff
@@ -36,10 +38,12 @@
                       <a href="#edit-in-place" class="field-label edit-in-place"><%= config.label %></a>
                       <%= sort.hidden_field :label, {data: {:"edit-field-target" => "true"}} %>
                     </h3>
-                    (<%= translate_sort_fields(config) %>)
                     <%= sort.hidden_field :weight, {value: index, data: {property: "weight"}} %>
                   <% end %>
                 </div>
+                <div class="card-body bg-white">
```

Comment:
I explicitly added this `bg-white` class since the `dd3-content` class got moved up (otherwise this background was gray-ish as well).  Happy to 🔪 this, but this kept the original design.